### DISCUSSION TITLE
how to enable push notification when alarm trigger?

### DISCUSSION THREAD

**Discussion by ghost** [taligentx/dscKeybusInterface#139] [dscKeybusInterface Discussion]
hi, how i can enable push notification when alarm trigger? only on display wrote that alarm triggered, but i cannot assign it to virtual pin!
best regards

**Comment by taligentx** [comment]
Hi @connected201 - what type of push notification service are you using?  The built-in example sketches demonstrate how to do this with PushBullet.

**Comment by onlineapps-cloud** [comment]
Hi, i cannot catch alarm signal to trigger push notification!

**Comment by taligentx** [comment]
Hi @onlineapps-cloud - which DSC panel do you have, and what is the output of the `Status` sketch when you trigger an alarm?
